kolibri/plugins/device/assets/src/views/DeviceSettingsPage/index.vue

```diff
@@ -560,7 +560,7 @@
     },
     created() {
       this.setDeviceURLs();
-      this.setFreeSpace();
+      if (this.freeSpace === 0) this.setFreeSpace();
```

Comment:
I can imagine a situation where the freeSpace value is actually 0 - i.e. when there is no free space on the device. Perhaps it would be better if this were initialized as `null` rather than `0` to be able to do a check on whether it has been set or not (would have to make sure anywhere it was being used was being guarded against the non-numeric value).

Reply:
I have updated the changes